### PR TITLE
Update entrypoint-sentinel.sh

### PR DESCRIPTION
**Pull request by iva3682**
add SENTINEL_ID for fixed sentinel myid

### CODE REVIEW

**Comment by drivebyer** [comment]
Why do we need custom this param ? @iva3682

**Comment by iva3682** [comment]
When you restart a pod with Sentinel, its myid changes to a random one. Other Sentinels receive his new myid, but the old one also takes part in the elections, which prevents the correct determination of the Sentinel master. Using the SENTINEL_ID parameter, you can fix myid to a specific value.
For example, (when using Stateful Sets) like this:
```
...
- env:
  - name: SENTINEL_ID
    valueFrom:
      fieldRef:
	apiVersion: v1
	fieldPath: metadata.name
...
```

**Comment by drivebyer** [comment]
I understand. It seems to be related to removing the sentinel in the cluster, as described in [this Redis documentation](https://redis.io/docs/latest/operate/oss_and_stack/management/sentinel/). I'm not sure if it works, because I couldn't find an example of sentinel myid in the [Redis GitHub repository](https://github.com/redis/redis/blob/unstable/sentinel.conf).

**Comment by iva3682** [comment]
Yes, for some reason Redis developers do not mention the possible "sentinel myid" configuration in sentinel.conf (it might make sense to create a corresponding issue in the redis repository), but this parameter is described here: https://github.com/redis/redis/blob/617909e943be0691ac9179aa1e6664937a4ca255/src/sentinel.c#L510. I use this patch in my cluster and it works fine: after restarting the Sentinel pod, the expected number of instances participate in the elections, not +1. There is no need to execute the SENTINEL RESET command after restarting the pod, this has a good effect on the autonomy of the Sentinel cluster.

**Comment by drivebyer** [comment]
Please fix https://github.com/OT-CONTAINER-KIT/redis/pull/73/checks?check_run_id=21976287248

File: entrypoint-sentinel.sh [hunk 1]
```diff
@@ -24,6 +24,9 @@ sentinel_mode_setup(){
     if [[ -n "${MASTER_PASSWORD}" ]];then
       echo "sentinel auth-pass ${MASTER_GROUP_NAME} ${MASTER_PASSWORD}"
     fi
+    if [[ -n "${SENTINEL_ID}" ]];then
```
Comment (drivebyer) [inline entrypoint-sentinel.sh:27]: https://github.com/OT-CONTAINER-KIT/redis/actions/runs/10935791553/job/30358796803?pr=73

**Pull request merged**